pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -74,23 +74,36 @@ const (
 
 const (
 	cephOSDStart = `
-function sigterm() {
-	echo "SIGTERM received"
+RESTART_INTERVAL=%d
+child_pid=""
+
+function handle_signal() {
+	echo "Signal received"
+	# kill the child process is its still running
```

Comment:
Needs typo correction "# kill the child process is its still running" should be "# kill the child process if its still running"